backend/courses/views.py

```diff
@@ -16,8 +16,9 @@
     StatusUpdateSerializer,
     UserSerializer,
 )
-from PennCourses.docs_settings import PcxAutoSchema
-from plan.search import TypedSectionSearchBackend
+from PennCourses.docs_settings import PcxAutoSchema, reverse_func
+from plan.filters import CourseSearchFilterBackend
+from plan.search import TypedCourseSearchBackend, TypedSectionSearchBackend
```

Comment:
I don't think we want to have `courses` depending on `plan` and `plan` depending on `courses`, it's not a strict circular dependency issue, but more a question of separation of concerns... But on the other hand, we do have some mentions of `review` all over. Not sure what the common practice is, if we should care that our apps are referencing each other or not.